alf/algorithms/algorithm.py

```diff
@@ -62,6 +62,7 @@ def _flatten_module(module):
         return [module]
 
 
+@alf.configurable(whitelist=['checkpoint_preload_strict'])
```

Comment:
I feel it is better to pass this argument from the subclass algorithms and config for the subclass algorithm so we don't make global change for the behavior of Algorithm.

Reply:
> I feel it is better to pass this argument from the subclass algorithms and config for the subclass algorithm so we don't make global change for the behavior of Algorithm.

I think the question is down to which subalgorithm? If all the way down to each leaf algorithm, there will be lots of arg change lines across the entire repo. 

Or I can add this option to TrainerConfig so that it's easier for the change.

Reply:
> > I feel it is better to pass this argument from the subclass algorithms and config for the subclass algorithm so we don't make global change for the behavior of Algorithm.
> 
> I think the question is down to which subalgorithm? If all the way down to each leaf algorithm, there will be lots of argument change lines across the entire repo.

Just change the specific algorithm that needs this behavior at this moment.